### PR TITLE
create the basic structure of To Do List

### PR DESCRIPTION
**Pull request by RashidMenhas**
create to do list.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Rashid, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an appropriate ReadMe. Kindly ensure that your ReadMe is detailed and reflects your project. I suggest you use this [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly request for another review when fix is done.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/cruds.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  return arrayobj;
+};
+
+export const removetask = (id) => {
+  arrayobj = arrayobj.filter((item) => `${item.index}` !== id);
+  arrayobj = arrayobj.map((item, index) => ({
+    ...item, index,
+  }));
+  localStorage.setItem('tasks', JSON.stringify(arrayobj));
+};
```
Comment (Wes-Isaac) [inline src/cruds.js]: - [ ] Great job implementing the required functions, however, after editing the change is not stored in local storage kindly fix that.

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+
```
Comment (Wes-Isaac) [inline README.md]: - [ ] for readme please use the [template](https://github.com/microverseinc/readme-template) provided 👍.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @RashidMenhas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Clean code 👍.
- Most requirements are completed 🚀.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/cruds.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+let arrayobj = [];
+export const addlist = (value) => {
```
Comment (Ismailco) [inline src/cruds.js:2]: - [ ] I can't add a task to the list or local storage, as the previous reviewer mentioned kindly make sure to fix this so your app works properly

Peek 2022-10-15 16-22

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @RashidMenhas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @RashidMenhas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**